### PR TITLE
CI: Unstable Rook integration tests that fail intermittently.

### PR DESCRIPTION
**Pull request by rajatsingh25aug**
[test full]
[test ceph]

**Description of your changes:**
There are a couple of integration tests that fail intermittently. Since these failures were very difficult to reproduce locally, hence opening a PR so that we can check the failure of those on the CI.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

**Comment by rajatsingh25aug** [comment]
27 tests failed, strange.
Pushing again.

File: tests/framework/installer/ceph_installer.go [hunk 1]
```diff
@@ -534,8 +534,12 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 		"objectbuckets.objectbucket.io",
 		"objectbucketclaims.objectbucket.io",
 		"cephrbdmirrors.ceph.rook.io")
-	checkError(h.T(), err, "cannot delete CRDs")
 
+	checkError(h.T(), err, "cannot delete CRDs")
+	logger.Infof("--------Now wil check if CRD are deleted or not------")
+	if err != nil {
+		logger.Infof("----Cannot delete CRDs-----")
```
Comment (travisn) [inline tests/framework/installer/ceph_installer.go:541]: Some suggestions to try when `err != nil`:
- kubectl get cephcluster <name> -o yaml // see if there are existing cephcluster CRs with finalizers that are preventing the deletion of the CRD
- If the cephcluster CR still exists, force remove the finalizer. Retry this several times if it keeps failing
- Wait and retry the DeleteResource multiple times if it fails

**Comment by rajatsingh25aug** [comment]
Thanks for the tips, will try that!.

**Comment by travisn** [comment]
How about if we close this for now since no specific action item was found yet

**Pull request closed without merging**